front/migrations/db/migration_138.sql

```diff
@@ -0,0 +1,37 @@
+-- Migration created on Dec 31, 2024
+ALTER TABLE "user_metadata" ALTER COLUMN "userId" SET NOT NULL;ALTER TABLE "user_metadata"  ADD FOREIGN KEY ("userId") REFERENCES "users" ("id") ON DELETE RESTRICT ON UPDATE CASCADE;
```

Comment:
Did you test this? IIRC this won't work as we need to delete the exiting ones.

Reply:
It runs

Reply:
It does create new constraints

Reply:
Will just remove it